ee/ucp/authorization/create-teams-with-ldap.md

```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
 ### Match Group Members (Direct Bind)
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
 synced to match the membership of the group.
 
-- **Group DN**: The distinguished name of the group from which to select users.
-- **Group Member Attribute**: The value of this group attribute corresponds to
+- **Group DN:** The distinguished name (DN) of the group from which to select users.
+- **Group Member Attribute:** The value of this group attribute corresponds to
   the distinguished names of the members of the group.
 
-
 ### Match Search Results (Search Bind)
 
 This option specifies that team members should be synced using a search query
 against your organization's LDAP directory. The team's membership will be
 synced to match the users in the search results.
```

Comment:
Suggest editing the first sentence to read: "The Search Bind option specifies that team members be synced using a search query against the organization's LDAP directory."